CMakeLists.txt

```diff
@@ -283,8 +287,14 @@ if(BUILD_TESTING)
       GTEST_HAS_TR1_TUPLE=0
       GTEST_USE_OWN_TR1_TUPLE=0)
   target_link_libraries(
-      UnitTest testutils GTest::main jaegertracing-static ${LIBS})
+      UnitTest testutils GTest::main jaegertracing ${LIBS})
   add_test(NAME UnitTest COMMAND UnitTest)
+
+  add_executable(DynamicallyLoadTracerTest src/jaegertracing/DynamicallyLoadTracerTest.cpp)
+  target_link_libraries(
+    DynamicallyLoadTracerTest OpenTracing::opentracing-static)
+  add_test(NAME DynamicallyLoadTracerTest COMMAND DynamicallyLoadTracerTest 
+           ${CMAKE_BINARY_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}jaegertracing${CMAKE_SHARED_LIBRARY_SUFFIX})
```

Comment:
Can you change `${CMAKE_BINARY_DIR}` with `${CMAKE_CURRENT_BINARY_DIR}`. I think it is better for recursive builds.

Reply:
Done.